src/transformers/pipelines/__init__.py

```diff
@@ -189,6 +193,19 @@
         },
         "type": "text",
     },
+    "visual-question-answering": {
+        "impl": VisualQuestionAnsweringPipeline,
+        "pt": (AutoModelForVisualQuestionAnswering,) if is_torch_available() else (),
+        "tf": (),
+        "default": {
+            "model": {
+                "pt": "dandelin/vilt-b32-finetuned-vqa",
+                "tokenizer": "dandelin/vilt-b32-finetuned-vqa",
+                "feature_extractor": "dandelin/vilt-b32-finetuned-vqa",
```

Comment:
In some way we already have community models used in the pipelines (`dbmdz` for the token classification), but as it's a central component that's likely to be the first component used by users, I wonder if we shouldn't push to have models maintained by HF to ensure these do not break. WDYT @patrickvonplaten @sgugger?

Reply:
Ideally, if we have a model available in the HF org, I'd prefer we use it, yes. If not, we can rely on a community contributed model as long the user understands the model can't be changed for BC reasons.

Reply:
Right, my worry is that if the model does change, then it's very likely to break many users' applications as it's the default model. To prevent such situations, maybe we could specify a revision for the default?

Reply:
I like the idea of using a revision, could we specific this somehow in the pipelines or do we need new logic for this?

Reply:
Yes revision is a good way to ensure it stays the same!

Reply:
Think for now, it's not really possible with the vanilla pipeline API - should we maybe handle this in a follow-up PR? Now we could merge as is and put up an issue "Add revision tag to VQA models (+ essentially all non-canonical models)" - wdyt @LysandreJik @sgugger ?

Reply:
Yes, sounds good to me.